### ISSUE TITLE
AFK Damage bypass using crystals & potions

### ISSUE THREAD

**Issue by bobhenl** [Zrips/CMI#7707]
### Description of Issue

Hi, the same way as there was an issue with the tp protection invulnerability, there's also a problem with AFK damage protection from the config. Players can bypass it with crystals & potions and may by other ways as well. Can it be fixed, please? 

### Version Information

```shell
[14:09:30 INFO]: --------------------------------------------------
[14:09:30 INFO]: CMI: 9.5.0.0 MySQL-> 9.5.0.0
[14:09:30 INFO]: CMILib: 1.2.5.2 
[14:09:30 INFO]: Server: Purpur 1.19.3-R0.1-SNAPSHOT
[14:09:30 INFO]: CMI economy: Enabled Vault: 1.7.3-CMI CMI Chat: Disabled 
[14:09:30 INFO]: Modules -> 45 enabled 12 disabled: deathMessages, disabledEnchants, customMessages, mirror, tablist, ranks, firstJoinMessages, playerCombat, votifier, skin, namePlates, playerChatTag
```

### Relevant Config Sections

```yaml
# Prevents damage while afk
  PreventDamage: true
```

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by bobhenl** [comment]
Yeah I've talked about end crystals, but may they can use other explosion blocks as well, like nether anchors etc.

**Comment by Zrips** [comment]
Tested with end crystals and damage isint being received while afk, as its causing generic damage. By any chance you have plugin which would control damage from blocks like this or maybe you have option to test this with CMI only to make sure that issues is with CMI and not 3rd party plugin causing secondary damage?
